docs/index.md

```diff
@@ -196,9 +196,25 @@ For more information, see the [settings documentation](settings.md).
 
 ### Running programmatically
 
-To run uvicorn directly from your application...
+There are many ways to run uvicorn directly from your application.
 
-**example.py**:
+#### `uvicorn.run` function
+
+The easiest way is to run via `uvicorn.run` function:
+
+```python
+import uvicorn
+
+async def app(scope, receive, send):
+    ...
+
+if __name__ == "__main__":
+    uvicorn.run("main:app", host="127.0.0.1", port=5000, log_level="info")
+```
```

Comment:
```markdown
`uvicorn.run` accepts the following arguments:

* `app`: Either an ASGI application, or (as shown above) an import string to the target ASGI application. The latter is required if using auto-reload or multiple workers.
* `**kwargs`: Keyword arguments that mirror command line options (see [Settings](/settings.md)): `--host <str>` becomes `host=<str>`, `--log-level <str>` becomes `log_level=<str>`, etc.
```

(I think this kind of info is valuable in docs, but maybe we might want to kick off an `API Reference` page for this kind of thing.)

Reply:
We _improved_ the `uvicorn.run` type annotation, so we can have autocomplete on it: https://github.com/encode/uvicorn/blob/b1a45821a2ab916b4428bdcb9eba4a4a0cc8fb74/uvicorn/main.py#L445-L492

I'll not apply this for now. An API reference would be fine by me.